Maven Compile Error

Hi @fbiville 
I went to try and run the tutorial you posted about in yor tweet which brought me to your fork of this repo and could not get it to package using 
mvn package -DskipTests

My maven version and jdk
```
mvn --version
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /usr/local/Cellar/maven/3.6.3/libexec
Java version: 14.0.1, vendor: AdoptOpenJDK, runtime: /Library/Java/JavaVirtualMachines/adoptopenjdk-14.jdk/Contents/Home
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.16", arch: "x86_64", family: "mac"
```


Here is the stack trace of the error I received:
```
[INFO] Using isolated classloader, without GMavenPlus classpath.
java.lang.NoClassDefFoundError: Could not initialize class org.codehaus.groovy.vmplugin.v7.Java7
	at org.codehaus.groovy.vmplugin.VMPluginFactory.<clinit>(VMPluginFactory.java:43)
	at org.codehaus.groovy.reflection.GroovyClassValueFactory.<clinit>(GroovyClassValueFactory.java:35)
	at org.codehaus.groovy.reflection.ClassInfo.<clinit>(ClassInfo.java:109)
	at org.codehaus.groovy.reflection.ReflectionCache.getCachedClass(ReflectionCache.java:95)
	at org.codehaus.groovy.reflection.ReflectionCache.<clinit>(ReflectionCache.java:39)
	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.registerMethods(MetaClassRegistryImpl.java:209)
	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.<init>(MetaClassRegistryImpl.java:107)
	at org.codehaus.groovy.runtime.metaclass.MetaClassRegistryImpl.<init>(MetaClassRegistryImpl.java:85)
	at groovy.lang.GroovySystem.<clinit>(GroovySystem.java:36)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:427)
	at org.codehaus.gmavenplus.util.ClassWrangler.getClass(ClassWrangler.java:303)
	at org.codehaus.gmavenplus.util.ClassWrangler.getGroovyVersionString(ClassWrangler.java:97)
	at org.codehaus.gmavenplus.util.ClassWrangler.logGroovyVersion(ClassWrangler.java:274)
	at org.codehaus.gmavenplus.mojo.AbstractCompileMojo.doCompile(AbstractCompileMojo.java:291)
	at org.codehaus.gmavenplus.mojo.CompileTestsMojo.execute(CompileTestsMojo.java:76)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:957)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:289)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:193)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  27.817 s
[INFO] Finished at: 2021-03-04T22:17:20-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.gmavenplus:gmavenplus-plugin:1.11.0:compileTests (default) on project liquibase-neo4j: Execution default of goal org.codehaus.gmavenplus:gmavenplus-plugin:1.11.0:compileTests failed: A required class was missing while executing org.codehaus.gmavenplus:gmavenplus-plugin:1.11.0:compileTests: Could not initialize class org.codehaus.groovy.reflection.ReflectionCache
[ERROR] -----------------------------------------------------
[ERROR] realm =    plugin>org.codehaus.gmavenplus:gmavenplus-plugin:1.11.0
[ERROR] strategy = org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy
```

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -14,6 +14,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        java_version:
+          - 8
+          - 11
         os: [ ubuntu-latest ]
         neo4j_version:
           - "3.5"
@@ -26,10 +29,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 8
+    - name: Set up JDK ${{ matrix.java_version }}
       uses: actions/setup-java@v1
       with:
-        java-version: 8
+        java-version: ${{ matrix.java_version }}
     - name: Cache local Maven repository
       uses: actions/cache@v2
       with:

--- a/README.md
+++ b/README.md
@@ -55,7 +55,7 @@ Running Java under /usr/local/Cellar/openjdk/15.0.1/libexec/openjdk.jdk/Contents
 Then, you need to get the JAR file of this plugin.
 One way to do so is to build from sources, which requires:
 
- - a [JDK 8 or later](https://openjdk.java.net/install/)
+ - a [JDK 8 or 11](https://openjdk.java.net/install/)
  - [Apache Maven](https://maven.apache.org/download.cgi)
 
 Once the repository is cloned, you can run:

